### ISSUE TITLE
Add command-line program for spitting out license etc

### ISSUE THREAD

**Issue by neilb** [Perl-Toolchain-Gang/Software-License#45]
More than once (hey, I'm getting old) I've gone to `S::L` looking for a command-line script that will spit out a license according to various options.

I end up finding [App::Software::License](https://metacpan.org/pod/App::Software::License) of course, but I think that such a program should be part of this dist. That uses `Moose` though, so in its current form it probably can't be dropped here, but how about a Mooified version?

**Comment by haarg** [comment]
There's already requests to reduce the number of dependencies this module has, so I don't think using Moo would be a great option.  It would be pretty trivial to write an app just using Getopt::Long directly though.

**Comment by neilb** [comment]
If we spec it out in these comments, maybe this module might get assigned to someone in the PRC for December, you never know ...

**Comment by Leont** [comment]
> There's already requests to reduce the number of dependencies this module has, so I don't think using Moo would be a great option. It would be pretty trivial to write an app just using Getopt::Long directly though.

Agreed. Such a tool would be useful, but can be written fairly minimalistic.

**Comment by karenetheridge** [comment]
A patch converting App::Software::License to away from Moose would be very thankfully received; there is a wishlist ticket in its queue already for that.  We can certainly look at merging that code into Software-License as well, once the dependencies have shrunk.

**Comment by Leont** [comment]
> A patch converting App::Software::License to away from Moose would be very thankfully received; there is a wishlist ticket in its queue already for that. We can certainly look at merging that code into Software-License as well, once the dependencies have shrunk.

Ironically, I suspect it's the configuration management (Config::Any) that's harder to get rid of than the Moose dependency.
